packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -125,7 +125,7 @@ describe('format()', () => {
       '日本',
       '〒100-8994東京都目黒区八重洲1-5-3',
       'Shopify',
-      '田中恵子様',
+      '田中 恵子様',
```

Comment:
That's because of fixture update. Now, we put a space between `firstName` and `lastName` in `ja`